app/component/itinerary/BicycleLeg.js

```diff
@@ -154,6 +175,58 @@ export default function BicycleLeg(
   const destination = bicycleWalkLeg?.to.stop
     ? bicycleWalkLeg?.to.name
     : leg.to.name;
+
+  async function makeNearestScooterQuery(from) {
+    setNearestScooterState({ loading: LOADSTATE.LOADING });
+    const planParams = {
+      lat: from.lat,
+      lon: from.lon,
+      maxResults: config.cityBike.maxNearbyRentalVehicleAmount,
+      first: config.cityBike.maxNearbyRentalVehicleAmount,
+      maxDistance: config.cityBike.maxDistanceToRentalVehiclesInMeters,
+      filterByModes: ['SCOOTER'],
+      filterByPlaceTypes: ['VEHICLE_RENT'],
+    };
+
+    const result = await fetchQuery(
```

Comment:
I guess there should be try-catch around this await block, to handle IO errors properly.